test/index.js

```diff
@@ -77,4 +77,57 @@ describe('validate', () => {
       assert.equal(err.message, 'message1')
     }
   })
+
+  it('should invoke checks with correct path', async () => {
```

Comment:
Rephrase "should invoke checks for nested path". Then add 2 fields: a.b (2 levels) and a.b.c (3 levels). 

If this test passes, no need for other tests because they should already have been covered by existing tests/conditions.